alica_customer_library/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
One kind of big thing to think about:  I don't like having a useless dummy package just to test this feature.  However, something that we WILL need is a *real* library of plans/behaviors/conditions that are useful in various situations.  So it would be good to start thinking of, at least, a couple behaviors that are somewhat widely applicable.  Plans are a bit harder without having template plans available.

We don't have to introduce a full useful library in this PR but at least the naming of the package should reflect that this is intended to be a useful default library rather than a dummy package